commons/es_client.py

```diff
@@ -188,3 +185,54 @@ def search_logs(self, search_query):
                 }
             }
         })
+
+    def search_logs_by_pattern(self, search_query):
+        return self.get_logs_by_query(search_query["project"], {
+            "size": 1000,
+            "query": {
+                "regexp": {
+                    "log_message": {
+                        "value": search_query["query"],
+                    }
+                }
+            }
+        })
+
+    def index_logs(self, index_query):
+        logs = index_query["logs"]
+        project_id = index_query["project"]
+        index_name = self.format_index_name(project_id)
```

Comment:
when index was not created you need to create on your own and with index template, and let's create a standard policy to rollover after 1 week and move to the warm phase after 2 weeks, but delete will be added only when update policy is called. So you need to add logic here with templates and policies.

Reply:
done